scripts/edit-task.js

```diff
@@ -0,0 +1,19 @@
+import { CLICK_EVENT } from "/scripts/constants.js";
+import onEditButtonClick from "/scripts//events/onEditButtonClick.js";
```

Comment:
incorrect import path.

```suggestion
import onEditButtonClick from "/scripts/events/onEditButtonClick.js";
```